metricbeat/module/elasticsearch/ccr/ccr.go

```diff
@@ -82,7 +82,7 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 	if ccrUnavailableMessage != "" {
 		if time.Since(m.lastCCRLicenseMessageTimestamp) > 1*time.Minute {
 			m.lastCCRLicenseMessageTimestamp = time.Now()
-			m.Logger().Warn(ccrUnavailableMessage)
+			m.Logger().Debug(ccrUnavailableMessage)
```

Comment:
I made this change for two reasons:

1. It's consistent with a similar logging behavior seen in the `elasticsearch/enrich` metricset: https://github.com/elastic/beats/blob/ee5a4e73a98fbf63307cd5d7f6cfcf196bb15bd3/metricbeat/module/elasticsearch/enrich/enrich.go#L82

2. By changing the log level to debug here (instead of warn), this check in the system test doesn't fail: https://github.com/elastic/beats/pull/14591/files#diff-a911af4a9815bc393395ac21029772e5R144